backend/data/src/main/kotlin/io/tolgee/service/dataImport/processors/FileProcessorContext.kt

```diff
@@ -86,6 +87,13 @@ data class FileProcessorContext(
         _translations[keyName]!!.removeIf { it.language == language && !it.isPlural }
       }
       _translations[keyName]!!.add(entity)
+
+      val key = getOrCreateKey(keyName)
```

Comment:
The generic data storing implementation should not be touch by adding new format. There already is a mechanism for adding description. Please use that:

io.tolgee.service.dataImport.processors.FileProcessorContext#addKeyDescription